static/src/javascripts-legacy/projects/common/modules/identity/delete-account.js

```diff
@@ -5,10 +5,19 @@ define([
     $,
     bean
 ) {
+    function disableDeleteButton() {
+        $('#deleteButton')[0] && ($('#deleteButton')[0].disabled = true);
+    }
+
+    function showLoader() {
+        $('#deleteLoader')[0] && $('#deleteLoader')[0].classList.remove("is-hidden");
+    }
+
     function setupLoadingAnimation() {
-        if ($('#deleteForm').length && $('#deleteLoader').length) {
+        if ($('#deleteForm')[0] && $('#deleteLoader')[0]) {
             bean.on($('#deleteForm')[0], 'submit', function() {
-                $('#deleteLoader')[0].classList.remove("is-hidden");
+                disableDeleteButton();
```

Comment:
I think we should activate the button again after some time (in case the request failed, because of connectivity problems) or in case the submit wasn't successful ..

I'd also vote for saving these selectors as a variable before, since this is very repetitive code ...